src-electron/db/query-attribute.js

```diff
@@ -164,23 +198,68 @@ async function selectAllAttributeDetailsFromEnabledClusters(
     ATTRIBUTE.NAME,
     ATTRIBUTE.CODE,
     ATTRIBUTE.SIDE,
-    ATTRIBUTE.TYPE,
+    CASE
+       WHEN
+         ATTRIBUTE.ARRAY_TYPE IS NULL
+       THEN
+         ATTRIBUTE.TYPE
+       ELSE
+         ATTRIBUTE.ARRAY_TYPE
+     END AS TYPE,
```

Comment:
This seems like a pretty significant behavior change.  Are there no existing consumers of this?

Reply:
This should be fine because entryType was introduced in matter xml and didn't exist in the old Zigbee 3.0 xml.

Reply:
And there are no consumers who are using this with Matter XML right now?

Reply:
Yeah this was not being used by Matter before.